e2e/helper_test.go

```diff
@@ -120,11 +126,6 @@ func commonAfterEach(cc CleanupContext) {
 				return fmt.Errorf("lvm num mismatched. before: %d, after: %d", cc.LvmCount, lvmCountAfter)
 			}
 
-			_, _, err = kubectl("get", "node", "-o", "json")
```

Comment:
Because getting node is done in getNodeAnnotationMapWithPrefix, it is not needed here.